sqlglot/parser.py

```diff
@@ -1450,6 +1457,42 @@ def _parse_exists(self, not_: bool = False) -> t.Optional[bool]:
             and self._match(TokenType.EXISTS)
         )
 
+    def _parse_function_body(self, create_token: Token, extend_props: t.Callable):
```

Comment:
Need a return type hint here.

Reply:
This is ok if we just want to factor out the UDF parsing and reduce noise in `_parse_create`, but I'm not sure about the split between stored procedures and functions. See my other comment in bigquery.py about this.